### PR TITLE
Support `referrerpolicy` on `amp-iframe`

### PR DESCRIPTION
**Pull request by aghassemi**
Closes https://github.com/ampproject/amphtml/issues/3950

### CODE REVIEW

**Comment by cramforce** [comment]
LGTM :)

**Comment by aghassemi** [comment]
@cramforce Added validator changes. Can you please PTAL and merge? 
/cc @Gregable

File: extensions/amp-iframe/0.1/validator-amp-iframe.protoascii [hunk 1]
```diff
@@ -53,6 +53,7 @@ tags: {  # <amp-iframe>
   attrs: { name: "allowfullscreen" value: "" }
   attrs: { name: "allowtransparency" value: "" }
   attrs: { name: "frameborder" value_regex: "0|1" }
+  attrs: { name: "referrerpolicy" value_regex: "no-referrer|origin|unsafe-url" }
```
Comment (cramforce) [inline extensions/amp-iframe/0.1/validator-amp-iframe.protoascii]: There are more possible values. I think we can allow all strings, though.

Comment (aghassemi) [inline extensions/amp-iframe/0.1/validator-amp-iframe.protoascii]: done

**Pull request merged**